db.json

```
@@ -0,0 +1,65 @@
+{
+    "productList": [
```

```suggestion
    "products": [
```
Để ở dạng số nhiều nhé